sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1500,31 +1537,35 @@ object functions {
   }
 
   /**
-   * Computes the cosine of the given value. Units in radians.
+   * @param e angle in radians
+   * @return  cosine of the angle, as if computed by [[java.lang.Math#cos]]
    *
    * @group math_funcs
    * @since 1.4.0
    */
   def cos(e: Column): Column = withExpr { Cos(e.expr) }
 
   /**
-   * Computes the cosine of the given column.
+   * @param columnName angle in radians
+   * @return           cosine of the angle, as if computed by [[java.lang.Math#cos]]
```

Comment:
You don't need to line these up, especially if it makes it extend past the line limit or wrap.